### PR TITLE
test: Automate `dwallet transfer` test by creating a DWallet and using it

### PR DESCRIPTION
**Pull request by ItayLevyOfficial**

### CODE REVIEW

File: sdk/typescript/test/e2e/signature-mpc.test.ts [hunk 1]
```diff
@@ -106,17 +106,11 @@ describe('Test key share transfer', () => {
...
-
-		let parsedKeyshare = Uint8Array.from(Buffer.from(secretKeyshare, 'hex'));
-
-		const encryptedUserShareAndProof = generate_proof(
-			parsedKeyshare,
-			recipientData?.encryptionKey!,
-		);
+		const dwallet = await createDWallet(toolbox.keypair, toolbox.client);
+		const dwalletID = dwallet?.dwalletId!;
+		let secretShareBytes = dwallet?.dkgOutput!.slice(0, 32);
```
Comment (zeevmoney) [inline sdk/typescript/test/e2e/signature-mpc.test.ts:111]: add some comment here, so we know why you took the first 32 bytes.

File: sdk/typescript/test/e2e/signature-mpc.test.ts [hunk 2]
```diff
@@ -133,7 +127,7 @@ describe('Test key share transfer', () => {
 		);
 
 		let secretUserShare = new Uint8Array(256);
-		secretUserShare.set(Uint8Array.from(Buffer.from(secretKeyshare, 'hex')).reverse());
+		secretUserShare.set(secretShare.reverse());
```
Comment (zeevmoney) [inline sdk/typescript/test/e2e/signature-mpc.test.ts:130]: why do you need the reverse?

**Review by zeevmoney — COMMENTED** [review]
see comments

File: sdk/typescript/test/e2e/signature-mpc.test.ts (hunk 2 above)
Comment (ItayLevyOfficial) [reply sdk/typescript/test/e2e/signature-mpc.test.ts:130]: idk tbh it was in the original code @yaelAbergel

**Review by zeevmoney — APPROVED** [review]

**Pull request merged**